tests/fsharp/Compiler/CodeGen/LiteralValue.fs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.UnitTests.CodeGen.EmittedIL
+
+open FSharp.Compiler.UnitTests
+open NUnit.Framework
+
+[<TestFixture>]
+module ``Literal Value`` =
+
+    [<Test>]
+    let ``Literal Value``() =
+        CompilerAssert.CompileLibraryAndVerifyIL
+            """
+module LiteralValue
```

Comment:
@TIHan I have to add `module` line and rename `arg` to `_` to make this test works.
It is OK that `CompilerAssert.CompileLibraryAndVerifyIL` require source code to compile without warnings?